os/services/orchestra/orchestra-rule-unicast-per-neighbor-rpl-storing.c

```diff
@@ -74,6 +74,17 @@ get_node_timeslot(const linkaddr_t *addr)
   }
 }
 /*---------------------------------------------------------------------------*/
+static uint16_t
+get_node_channel_offset(const linkaddr_t *addr)
+{
+  if(addr != NULL && ORCHESTRA_UNICAST_MAX_CHANNEL_OFFSET > ORCHESTRA_UNICAST_MIN_CHANNEL_OFFSET) {
```

Comment:
shouldn't we allow `ORCHESTRA_UNICAST_MAX_CHANNEL_OFFSET == ORCHESTRA_UNICAST_MAX_CHANNEL_OFFSET` ?

Reply:
True that. Updated now...